### ISSUE TITLE
Mapbox "Image Server Access Check" not working

### ISSUE THREAD

**Issue by AnUntitledAccount** [derekhe/msfs2020-map-enhancement#60]
Hi,
I've created a free mapbox account and copied the default public access token to the program. However, the cross keeps coming up for "Image Server Access Check".  I've tried restarting the app, restarting MFS and resetting the map enhancement app to defaults. However, the problem still persists. Is there a way to fix this? 

Logs: 
[error.log](https://github.com/derekhe/msfs2020-map-enhancement/files/8057667/error.log)
[localgoogle-access.log](https://github.com/derekhe/msfs2020-map-enhancement/files/8057671/localgoogle-access.log)
[khstore-access.log](https://github.com/derekhe/msfs2020-map-enhancement/files/8057670/khstore-access.log)
[virtualearth-access.log](https://github.com/derekhe/msfs2020-map-enhancement/files/8057672/virtualearth-access.log)

![image](https://user-images.githubusercontent.com/50532251/153794929-12344d5c-2c58-4dca-bfff-af278cc812e3.png)
![image](https://user-images.githubusercontent.com/50532251/153795552-92834ab2-e678-4f10-b0d0-7b9e25e28411.png)
![image](https://user-images.githubusercontent.com/50532251/153795526-7f27a9a7-712a-4003-a54b-4edad903cb91.png)

**Comment by h0wardch3ng** [comment]
```

[2022-02-14 22:38:03.249] [info]   * Running on http://127.0.0.1:39871/ (Press CTRL+C to quit)

[2022-02-14 22:38:12.951] [info]  [2022-02-14 22:38:12,950] ERROR in app: Exception on /tiles/akh12101.jpeg [GET]
Traceback (most recent call last):
  File "E:\Game\MME\MSFS2020 Map Enhancement\resources\extra\server\python\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "E:\Game\MME\MSFS2020 Map Enhancement\resources\extra\server\python\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "E:\Game\MME\MSFS2020 Map Enhancement\resources\extra\server\python\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "E:\Game\MME\MSFS2020 Map Enhancement\resources\extra\server\python\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "E:\Game\MME\MSFS2020 Map Enhancement\resources\extra\server\server.py", line 118, in tiles
    url = map_provider.map(quadkey)
  File "E:\Game\MME\MSFS2020 Map Enhancement\resources\extra\server\.\providers.py", line 60, in map
    return f"https://api.mapbox.com/v4/mapbox.satellite/{level_of_detail}/{tile_x}/{tile_y}.jpg?sku=cky8e1hd40jus15nzunvf7q4u&access_token={self.accessToken}"
AttributeError: 'MapBox' object has no attribute 'accessToken'

[2022-02-14 22:38:12.951] [info]  127.0.0.1 - - [14/Feb/2022 22:38:12] "GET /tiles/akh12101.jpeg?n=z&g=9580 HTTP/1.1" 500 -

[2022-02-14 22:38:14.009] [info]  [2022-02-14 22:38:14,008] ERROR in app: Exception on /tiles/akh12101.jpeg [GET]
Traceback (most recent call last):
  File "E:\Game\MME\MSFS2020 Map Enhancement\resources\extra\server\python\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "E:\Game\MME\MSFS2020 Map Enhancement\resources\extra\server\python\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "E:\Game\MME\MSFS2020 Map Enhancement\resources\extra\server\python\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "E:\Game\MME\MSFS2020 Map Enhancement\resources\extra\server\python\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "E:\Game\MME\MSFS2020 Map Enhancement\resources\extra\server\server.py", line 118, in tiles
    url = map_provider.map(quadkey)
  File "E:\Game\MME\MSFS2020 Map Enhancement\resources\extra\server\.\providers.py", line 60, in map
    return f"https://api.mapbox.com/v4/mapbox.satellite/{level_of_detail}/{tile_x}/{tile_y}.jpg?sku=cky8e1hd40jus15nzunvf7q4u&access_token={self.accessToken}"
AttributeError: 'MapBox' object has no attribute 'accessToken'

[2022-02-14 22:38:14.009] [info]  127.0.0.1 - - [14/Feb/2022 22:38:14] "GET /tiles/akh12101.jpeg?n=z&g=9580 HTTP/1.1" 500 -

[2022-02-14 22:38:16.045] [info]  [2022-02-14 22:38:16,044] ERROR in app: Exception on /tiles/akh12101.jpeg [GET]
Traceback (most recent call last):
  File "E:\Game\MME\MSFS2020 Map Enhancement\resources\extra\server\python\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "E:\Game\MME\MSFS2020 Map Enhancement\resources\extra\server\python\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "E:\Game\MME\MSFS2020 Map Enhancement\resources\extra\server\python\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "E:\Game\MME\MSFS2020 Map Enhancement\resources\extra\server\python\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "E:\Game\MME\MSFS2020 Map Enhancement\resources\extra\server\server.py", line 118, in tiles
    url = map_provider.map(quadkey)
  File "E:\Game\MME\MSFS2020 Map Enhancement\resources\extra\server\.\providers.py", line 60, in map
    return f"https://api.mapbox.com/v4/mapbox.satellite/{level_of_detail}/{tile_x}/{tile_y}.jpg?sku=cky8e1hd40jus15nzunvf7q4u&access_token={self.accessToken}"
AttributeError: 'MapBox' object has no attribute 'accessToken'
```

I guess the token field wasn't added correctly, the check is failed.

**Comment by AnUntitledAccount** [comment]
Ah ok, I thought I was doing something wrong. Hopefully this gets fixed in a new update.

**Comment by h0wardch3ng** [comment]
> Ah ok, I thought I was doing something wrong. Hopefully this gets fixed in a new update.

Switch to version 3.2.4 to avoid this bug.

**Comment by derekhe** [comment]
Fixed in 4.0.0, please try.
